e2e/Sandbox/DashboardCreators/AmazonAthenaDashboard.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Reveal.Sdk.Dom;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sandbox.DashboardFactories
+{
+    internal class AmazonAthenaDashboard : IDashboardCreator
+    {
+        public string Name => "Amazon Athena Data Source";
+
+        public RdashDocument CreateDashboard()
+        {
+            return null;

```

Comment:
instead of return null, throw an exception

Reply:
updated